### ISSUE TITLE
RDF bug.

### ISSUE THREAD

**Issue by rayday88** [dalaranwow/dalaran-wow#4847]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I que for a RDF on my hunter and it immediately says i've been in que for some ridiculous amount of hours(currently at 4hrs and 50mins).  It also states all its missing is one dps yet the q never pops. when i que'd with someone else it said the same thing at first but then it fixed and said 3 dps were in the que and gave the actual right amount of time we were in que for which was only 2 mins.
**Expected behaviour**: Tell us what should happen instead
It should say you've been in que for less than one minute not hours. and give a accurate representation of how many people are in que with you
**Steps to reproduce the problem**:

1. have a full group que rdf
2. complete dg then leave
3. que again by yourself after leaving the party. (this is all speculation but this is the last thing i did before the bug occured. this is also the only character it does it on.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_081617_012914](https://user-images.githubusercontent.com/31058098/29348533-740315ea-8222-11e7-87ca-0fd716752fc4.jpg)

**Comment by rayday88** [comment]
![wowscrnshot_081617_013306](https://user-images.githubusercontent.com/31058098/29348602-ea9bdb7e-8222-11e7-8254-f1a850e10fe8.jpg)
notice the time on the world clock

**Comment by Rushor** [comment]
The reason for this stucking is following:

If someone denies the proposal, you are not added again on top of the queue list.

This was fixed on PTR with this commit: 5f26f265170b4e201e66674a078fc2e44feb0970 from robinsch

Your issue posted here is a duplicate of issue: https://github.com/dalaranwow/dalaran-wow/issues/2331 (< click on this and you will be redirected to the original posted issue).
